.github/workflows/ci.yml

```
@@ -33,8 +33,81 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
+    - name: Set up date and time variable
+      run: echo "DATE_TIME=$(date '+%Y-%m-%d-%H-%M-%S')" >> $GITHUB_ENV 
+    - name: Output date and time
+      run: |
+        ls -all;pwd;whoami;
+        echo "Date and Time: ${{ env.DATE_TIME }}"
+    - name: Test build
+      run: |
+        #docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_PAT }} ghcr.io 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)